.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
```

Comment:
not sure if I'm picking nits but I'd start with more benevolent configuration: 90 days of inactivity and 14 days to close.

Reply:
OK, sounds reasonable.